src/shared/standalone-date-input/standalone-date-input.tsx

```diff
@@ -123,7 +123,7 @@ export const Component = (
     // EVENT HANDLERS
     // =============================================================================
     const handleSectionClick = () => {
-        if (disabled || readOnly) {
```

Comment:
Why is `readOnly` being removed?

Reply:
Thanks to catch this typo. it's not needed to remove anymore.

the isTouch was added not needed to remove the click action check some more.